nixos/modules/security/polkit.nix

```
@@ -12,11 +12,7 @@ in
 
   options = {
 
-    security.polkit.enable = mkOption {
-      type = types.bool;
-      default = true;
-      description = "Whether to enable PolKit.";
-    };
+    security.polkit.enable = mkEnableOption "enable PolKit";
```

```suggestion
    security.polkit.enable = mkEnableOption "PolKit";
```

https://github.com/NixOS/nixpkgs/blob/071dbeb56deeb56028486048fc9062213d92135d/lib/options.nix#L99

---

Thanks, fixed.